app/views/includes/head.njk

```diff
@@ -14,7 +14,13 @@
   ga('set', 'displayFeaturesTask', null);
   ga('set', 'transport', 'beacon');
 
-  ga('send', 'pageview');
```

Comment:
As far as I understand (https://developers.google.com/analytics/devguides/collection/analyticsjs/command-queue-reference#method-details), without `send` command we won't trace the traffic at all.

Reply:
Ah ok should it be 
```
ga('set', 'page', filtered_page_path);
ga('send', 'pageview');
```
then?

Reply:
I think so.

Reply:
the analytics consultant from Merkle says it isn't needed so let's go with what he says first.

Reply:
I don’t see the bit in the email where they say you dont need the send line 🤔 I could be wrong hard to read big threads

I wonder if we should keep it and have it like you say above
```
ga('set', 'page', filtered_page_path);
ga('send', 'pageview');
```